src/components/description_list/description_list_description.styles.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css } from '@emotion/react';
+import { euiFontSize, logicalTextAlignCSS } from '../../global_styling';
+import { UseEuiTheme } from '../../services';
+import { euiTitle } from '../title/title.styles';
+
+export const euiDescriptionListDescriptionStyles = (
+  euiThemeContext: UseEuiTheme
+) => {
+  const { euiTheme } = euiThemeContext;
+
+  return {
+    euiDescriptionList__description: undefined,
```

Comment:
```suggestion
    euiDescriptionList__description: css``,
```

The correct Emotion className/EUI snapshot name isn't being generated because of this - you'll need to rerender snapshots after this change